targets/rx5808.ini

```diff
@@ -60,3 +60,24 @@ build_flags =
 
 [env:RX5808_Diversity_ESP01F_Backpack_via_WIFI]
 extends = env:RX5808_Diversity_ESP01F_Backpack_via_UART
+
+[env:RX5808_HT_Backpack_via_UART]
+extends = env:RX5808_ESP01F_Backpack_via_UART
+build_flags =
+	${env:RX5808_ESP01F_Backpack_via_UART.build_flags}
+	-D HAS_HEADTRACKING
+	-D PIN_SDA=2
+	-D PIN_SCL=4
+	-D PIN_INT=9
```

Comment:
The ESP01F doesn't break out PIN 9 ?
![esp01f](https://github.com/user-attachments/assets/b1338d71-4996-41a4-99ed-3d1d006b3c0c)

Reply:
Indeed... sorted.